src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -568,4 +578,83 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
 
     return null;
   }
+
+  /**
+   * A {@link FlutterSdk} that is compatible with the Bazel version of the Dart and Flutter SDK.
+   */
+  public static class BazelSdk extends FlutterSdk {
+    /**
+     * The Bazel workspace for this project.
+     */
+    @NotNull final Workspace workspace;
+
+    @NotNull final Project project;
+
+    private BazelSdk(@NotNull Project project, @NotNull Workspace workspace) {
+      super(
+        Objects.requireNonNull(workspace.getRoot().findFileByRelativePath("mobile/flutter/tools/ide/gflutter")),
```

Comment:
Should we move these paths to `dart/config/intellij-plugins/flutter.json` (and loaded from io.flutter.bazel.Workspace.java`)?

Reply:
Yes.  Done.